quickstart/app/helper/function.php

```diff
@@ -0,0 +1,20 @@
+<?php
+function GetCategory($mang,$parent,$shift,$active)
```

Comment:
```suggestion
function getCategory($array, $parent, $shift, $active)
```
dùng tiếng anh nhé, tên hàm viết thường chữ cái đầu theo kiểu camelCase